scripts/index.js

```diff
@@ -31,12 +31,28 @@ const processFile = async (filePath) => {
     let detail = undefined;
     let infoArr = [];
 
-    for (let i = 1; i < lines.length; i++) {
+    let i = 0;
```

Comment:
I removed the extra `console.log`. 

Thanks for the suggestion! Let's do this in a future iteration of this script, which would need to be somewhat rewritten anyway, because there are also operator modifiers, formats etc. that would need to be completed. But I agree that some functional stuff could improve this.